dist/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap.min.css">
+</head>
+<body>
+    <div id="root"></div>
+<script type="text/javascript" src="./polyfills.min.js"></script><script type="text/javascript" src="./vendor.min.js"></script><script type="text/javascript" src="./main.min.js"></script></body>
```

Comment:
these file is generated programmatically, I've just forgotten to remove the dist folder from git